snippets/js.cson

```diff
@@ -23,3 +23,9 @@
             //      ${3:summary}
             ${1:name}: ${4:value},${5}
         """
+    'log the value of a variable':
+        'prefix': 'logvalue'
+        'body': """
+        console.log('${1:variable}', ${1:variable});
```

Comment:
what do you think about using debug? i think it makes it easier to highlight what you are doing since all of our methods use log. It makes the filter more useful in the dev tools also.